tests/functional/base_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2023.
```

Comment:
4

Reply:
:) i think we can update this in a followup but yes it should be 2024